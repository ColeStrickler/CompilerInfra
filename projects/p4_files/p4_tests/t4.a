

swag : custom {
    a : int;
    b : bool;

    swagyolo : (arg1: int, b: bool, c : int) -> bool
    {
        a = c * arg1;
        return a == 4;
    }
};

swag: int;