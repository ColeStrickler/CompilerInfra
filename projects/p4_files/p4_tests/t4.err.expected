FATAL [14,1]-[14,5]: Multiply declared identifier
Name Analysis Failed